modules/migration-prerequisites.adoc

```diff
@@ -24,6 +23,28 @@ endif::[]
 ** The clusters must be able to communicate using OpenShift routes on port 443.
 ** The clusters must have no critical conditions.
 ** The clusters must be in a ready state.
+** The appropriate firewall rules must be configured on the target cluster.
+** Load balancing must be correctly configured on the target cluster.
+
+* Resources:
+
+** Check whether your application uses a service network or an external route to communicate with services and make sure the relevant route exists.
+** If your application uses non-namespaced resources, you must re-create them on the target cluster.
+** If you are not using an external image registry, you must migrate images from the internal image registry. If they cannot be migrated, you must re-create them manually on the target cluster.
+** Remove old builds, deployments, and images from each namespace to be migrated using the prune command.
+** Exclude PVs, image streams, and other resources if you do not want to migrate them with the MTC.
+
+* Namespaces:
+
+
+** Check the namespaces on the target cluster to ensure that they do not duplicate namespaces to be migrated.
+** Do not create namespaces for your application on the target cluster before migration because this can change quotas.
```

Comment:
We do not use "can change" because it creates localization problems. It is too vague. See IBM style guide and rephrase. Also, be more specific about which quotas might be changed. An xref would be good. Now we can use them sparingly in modules.